.github/workflows/rust.yml

```diff
@@ -81,44 +81,3 @@ jobs:
           cargo test --target ${{ matrix.target }} --manifest-path $I/Cargo.toml
         done
 
-  publish-dry-run:
```

Comment:
Instead of removing the publish-dry-run block, which actually works fine most of the time but breaks in the edge case where a new crate is added, you can remove it from the complete job's `needs` list at the top of the yml.